Makefile

```diff
@@ -0,0 +1,33 @@
+LDFLAGS = -ldflags "-X main.gitSHA=$(shell git rev-parse HEAD)"
+
+OS := $(shell uname)
+
+build: clean
+	go build $(LDFLAGS) -o align
+
+test:
+	go test -v
+
+clean:
+	go clean
+	rm -f align
+	rm -f bin/*
```

Comment:
If you're wroing to write a binary to that directory.  If not, you can just remove the line from the makefile